assets/scss/components/_button.scss

```diff
@@ -0,0 +1,124 @@
+.button-title {
+  font-size: 22px;
+  font-weight: 400;
+  color: $white;
+  font-family: "";
+  src: url();
+}
+.button-wrapper {
+  height: 61px;
+  width: 190px;
+  position: relative;
+  overflow: hidden;
+  &:hover {
+    .main-button-change {
+      transform: translateY(0);
+    }
+  }
+  .main-button {
+    background: $primaryColor;
+    height: 100%;
+    width: 100%;
+    border: 0;
+    border-radius: 5%;
+  }
+  .main-button-change {
+    position: absolute;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    border-radius: 5%;
+    border: 0;
+    color: $primaryColor;
+    background: $darkGray;
+    transform: translateY(100%);
+    transition: 0.25s ease-in all;
+    outline: $white;
+  }
+}
+.button {
+  width: 200px;
+  height: 10px;
```

Comment:
Tutaj chyba coś nietak

Reply:
Poprawiłem usunięty został height i dodany padding by łatwiej było trafić w strzałkę.

Reply:
ale niestety tak jak pisałem na discordzie po usunięciu height nie da ustawić się transition na strzałkę podobny problem był w headerze w h1 tam poprawiłem to że działa ale tym sposobem co w headerze nie da się tego poprawić strzałka opada i się rozjeżdża. Gdy jest height to wszystko działa w porządku.